MAUI/DataGrid/Editing.md

```diff
@@ -0,0 +1,376 @@
+
+ * [RowColumnIndex](): Gets the current row and column index of the DataGrid.
+ * [Cancel](): When this member set to ‘true’, the event is canceled and the edited value is not committed in the underlying collection.
+
+To hook the `SfDataGrid.CurrentCellEndEdit` event, follow the code example:
+
+{% highlight c# %}
+this.dataGrid.CurrentCellEndEdit += DataGrid_CurrentCellEndEdit;
+
+private void DataGrid_CurrentCellEndEdit(object sender, DataGridCurrentCellEndEditEventArgs e)
+{
+    // Editing prevented for the cell at RowColumnIndex(1,3).
+    if (e.RowColumnIndex == new Syncfusion.Maui.GridCommon.ScrollAxis.RowColumnIndex(1, 3))
+        e.Cancel = true;
+}
+
+{% endhighlight %}
+
+## Programmatically edit a cell
+
+### Begin editing
+
+The SfDataGrid allows you to programmatically edit a cell by calling the [SfDataGrid.BeginEdit]() method. By calling this method, the particular cell enters into edit mode, allowing you to edit the data manually or programmatically. To programmatically edit a cell, refer to the following code example:
+
+{% highlight c# %}
+this.dataGrid.Loaded += DataGrid_Loaded;
+private void DataGrid_Loaded(object sender, EventArgs e)
+{
+    //Edit the cell at 2nd row,2nd column programmatically
+    this.dataGrid.BeginEdit(2, 2);
+}
+{% endhighlight %}
+
+### End editing
+
+The [SfDataGrid.EndEdit]() method can be called to programmatically end editing. When this method is called, a cell that is currently in edit mode will commit the edited value to the underlying collection and exit the edit mode. To programmatically end the editing process, refer to the following code example:
+
+{% highlight c# %}
+this.dataGrid.EndEdit();
+{% endhighlight %}
+
+### Cancel editing
+
+The [SfDataGrid.CancelEdit]() method can be called to programmatically cancel editing. When this method is called, a cell that is currently in edit mode will exit the edit mode without committing the edited value to the underlying collection. To programmatically cancel the editing process, refer to the following code example:
+
+{% highlight c# %}
+this.dataGrid.CancelEdit();
+{% endhighlight %}
+
+## How to
+
+### Cancel editing for a particular Cell
+
+The [SfDataGrid.CurrentCellBeginEdit]() event can be used to cancel the editing operation for the corresponding cell. To cancel the editing operation using the `SfDataGrid.CurrentCellBeginEdit` event, follow the code example:
+
+{% highlight c# %}
+this.dataGrid.CurrentCellBeginEdit += DataGrid_CurrentCellBeginEdit;
+private void DataGrid_CurrentCellBeginEdit(object sender, DataGridCurrentCellBeginEditEventArgs e)
+{
+    if (e.Column.MappingName == "OrderID" || e.RowColumnIndex.RowIndex == 2)
+        e.Cancel = true;
+}
+{% endhighlight %}
+
+### Cancel edited value from getting committed
+
+To prevent the edited value from getting committed, use the [CurrentCellEndEdit]() event. To prevent the edited values from getting committed in the underlying collection, follow the code example:
```

Comment:
To prevent the edited value from being committed, use the [CurrentCellEndEdit]() event. This event prevents the edited values from being committed to the underlying collection. Refer to the code example below:

Reply:
![image](https://github.com/syncfusion-content/maui-docs/assets/113881396/e351e3c8-004f-41db-bd58-eb6e50c31ff4)